man/systemd-creds.xml

```diff
@@ -354,7 +354,7 @@
       </varlistentry>
 
       <varlistentry>
-        <term><option>--tpm2-pcrs=</option><arg rep="repeat">PCR</arg></term>
+        <term><option>--tpm2-pcrs=</option><replaceable>PCR</replaceable></term>
```

Comment:
Here, `rep="repeat"` tried to indent that the option takes multiple values : `PCR1+PCR2+...`. But `<replaceable>` does not imply that.